### ISSUE TITLE
fix broken source:Crow Wing County, MN

### ISSUE THREAD

**Issue by justinelliotmeyers** [openaddresses/openaddresses#3198]
new source: https://gis.crowwing.us/cwcexternal/rest/services/CROWWINGPUBLIC/MapServer/4

fixes:http://results.openaddresses.io/sources/us/mn/crow_wing

**Comment by trescube** [comment]
I'm working on this.  Turns out that the `ADDRESS` field contains embedded units.

**Comment by justinelliotmeyers** [comment]
I was looking at buying land up here so I started to download the parcels - and somehow it was getting up to 4 million + locations (through a different way to grab rest service data) - checked oa and it was broken. anything I can do to help?

**Comment by trescube** [comment]
Nah, I just have to work out the regexes when I have a little more time.  Same with some of the other sources.  Great find, though!

**Comment by trescube** [comment]
I thought you were moving to Costa Rica?

**Comment by justinelliotmeyers** [comment]
we are - just diversifying my investments while we're stateside! We drove through there on our way home from Canada and there are a ton of lakes - would be perfect for the pontoon, dogs, etc

**Comment by trescube** [comment]
nice!
